src/core/ActiveStatus.cpp

```diff
@@ -396,7 +396,18 @@ void ActiveStatus::extendOneBlock() {
     mBlockArray.push_back(b);
 
     /* add to LRU cache */
-    mLRUCache->put(b.blockId, b.bucketId);
+    std::vector<int> blockIds = mLRUCache->put(b.blockId, b.bucketId);
```

Comment:
Which case will lead to this situation? I thought the acquireNewBlocks have already handled all cases.